parse/parse.go

```diff
@@ -59,6 +125,18 @@ func parseSpaceSeparatedDependency(scanner *bufio.Scanner, deps *types.ProjectLi
 	text := scanner.Text()
 	s := strings.Split(text, " ")
 	if criteria(s) {
-		deps.Projects = append(deps.Projects, types.Projects{Name: s[0], Version: s[1]})
+		if len(s) > 3 {
```

Comment:
yah 100% ¯\_(ツ)_/¯ atm i have no idea why this is. But also this is go.sum soooooo ☠️ 

https://github.com/sonatype-nexus-community/nancy/pull/155

Reply:
I did this BECAUSE if you have a `replace` block in your `go.mod`, `go list -m all` looks like this:

`golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2 => golang.org/x/crypto v0.0.0-20200604202706-70a84ac30bf9`

Kinda the best thing I could figure out.

Reply:
Maybe refactor this into something like this to make it nice and explicit. 

```
func isAReplaceBlock(s) bool {
  return len(s) > 3
}
```

Or since a replace line is actually this maybe just checking if that first entry is == "replace" then we do the different things.
`replace github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.2`

could the check just